### ISSUE TITLE
x509: certificate signed by unknown authority

### ISSUE THREAD

**Issue by sergioasantiago** [seznam/slo-exporter#44]
I'm using prometheusIngester module to query my prometheus instance. It is behind a web proxy that does the SSL termination. I'm getting the following issue when using the default Docker image from dockerhub.

```
slo-exporter_1  | time="2021-02-08T15:45:32.03454Z" level=error msg="failed querying Prometheus: 'Post "https://my-prometheus.com": x509: certificate signed by unknown authority'" component=prometheusIngester query="http_server_requests_seconds_count"
```

To fix that I need to manually install `ca-certificates`, ending up with the following Dockerfile

```
FROM seznam/slo-exporter:v6.7.0

RUN apt update && apt install ca-certificates -y
```

I can send a PR adding this line if you agree.

**Comment by FUSAKLA** [comment]
Hi, thanks for rising the issue! 
Your request sounds reasonable to me, so please feel free to send the PR :+1: 

Just please add also the `apt clean` after the installation to keep the image small as possible.
Thanks!

**Comment by david-vavra** [comment]
Merged, [released as v6.7.1](https://github.com/seznam/slo-exporter/releases).

Thank you for contributing.

Btw feel free to join us at [slack](https://join.slack.com/t/slo-exporter/shared_invite/zt-mnqxqv1s-1zaJtDiYbuVoOCCAMQi4Kg), in case you come across other issues, or just want to discuss use-cases you want to address with slo-exporter. :)
